components/layout_2020/fragment_tree/box_fragment.rs

```diff
@@ -190,6 +191,39 @@ impl BoxFragment {
         )
     }
 
+    pub fn clip_overflow(&self, containing_block_rect: &PhysicalRect<Au>) -> PhysicalRect<Au> {
+        // Area that the box is allowed to paint in.
+        let mut clip_rect = match self.style.get_box()._servo_overflow_clip_box {
```

Comment:
If you are enabling `overflow-clip-box-inline/block` then I would check these here.
Otherwise don't enable them.